src/css/profile/mobile/common/listview.less

```diff
@@ -274,7 +274,10 @@ tau-expandable {
 			justify-content: flex-start;
 			margin-top: 33 * @unit_base;
 			margin-bottom: 33 * @unit_base;
-			padding: 0 32 * @unit_base;
+			padding-top: 0;
```

Comment:
You set all padding directions. You can set simpler:
`padding: 0 24 * @px_base 0 24 * @px_base`

Reply:
done